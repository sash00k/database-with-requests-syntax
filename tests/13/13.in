get records where ((((name is "Кузнецов И. П." and course is set) or department is "ФФ") or course is set) or student_id is set)