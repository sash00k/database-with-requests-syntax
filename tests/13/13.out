Student Иванов С. В. from group 22129, ММФ
Student Петрова И. К. from group 19301, ФФ
Student Медведев Т. С. from group 19402, ФЕН
Student Зуброва Д. А. from group 8126, ММФ
Student Иванова В. В. from group 22129, ММФ
Student Кузнецов И. П. from group 8126, ММФ
Teacher Кузнецова Л. И. on course Программирование. Works with 2 groups and 3 students.
Teacher Кузнецов И. П. on course Программирование. Works with 2 groups and 0 students.